azkaban-common/src/main/java/azkaban/executor/container/KubernetesContainerizedImpl.java

```diff
@@ -1212,20 +1233,23 @@ private void deletePod(final int executionId) throws ExecutorManagerException {
   private void deleteService(final int executionId) throws ExecutorManagerException {
     final String serviceName = getServiceName(executionId);
     try {
-      final V1Status deleteResult = this.coreV1Api.deleteNamespacedService(
-          serviceName,
-          this.namespace,
-          null,
-          null,
-          null,
-          null,
-          null,
-          new V1DeleteOptions());
-      logger.info("ExecId: {}, Action: Service Deletion, Service Name: {}, code: {}, message: {}",
-          executionId,
-          serviceName,
-          deleteResult.getCode(),
-          deleteResult.getMessage());
+      // Using GenericKubernetesApi due to a Known issue in K8s Java client and OpenAPIv2:
+      // See more here: https://github.com/kubernetes-client/java/wiki/6.-Known-Issues
```

Comment:
Refer to this github issue directly in the comment: https://github.com/kubernetes-client/java/issues/86

The currently linked page will likely keep changing with different versions of the kubernetes client (for example if the issue actually gets fixed)